CHANGE-NOTES.html

```diff
@@ -1 +1 @@
-<p>Dropped support for IntelliJ 2020.3, 2021.1 and 2021.2.</p>
+<p>Dropped support for IntelliJ 2020.3, 2021.1, 2021.2 and  2021.3.</p>
```

Comment:
Pls add some _reassuring_ note that the already-released versions of the plugin will be available indefinitely on older IntelliJ versions

Reply:
:care: